src/client.ts

```diff
@@ -1779,7 +1779,7 @@ export class StreamChat<
         >,
     options: SearchOptions = {},
   ) {
-    // Return a list of channels
+    const { sort: sort_value, ...options_without_sort } = { ...options };
```

Comment:
Same as in channel but also we can probably avoid the rest spread as well, and pop everything to one line via 

```
      sort: sort &&= normalizeQuerySort(sort_value)
```

Reply:
I'm not quite sure what you want to do here. Which lines will be replaced with your suggestion?